src/libs/auth.js

```diff
@@ -93,8 +93,10 @@ window.forceSignIn = async token => {
 
 authStore.subscribe(async (state, oldState) => {
   if (!oldState.isSignedIn && state.isSignedIn) {
+    const userEmailString = state.user.email.match(/@.*/g).toString()
```

Comment:
This technically works, but it would be more typical to just index into the array with `[0]`, since that's the string we're after.